practices/appeng/adrs/0002-distributed-tracing.md

```diff
@@ -0,0 +1,156 @@
+# 0002 - Use OpenTelemetry to instrument code for distributed tracing
+
+**Status:** In Progress
+
+**Date Superseded:** (if applicable)
+
+**Date Accepted:**
+
+**Reviewers:** @jim @reggieriser @rogeruiz \<tag yourself!>
+
+## Context
+
+### Objective
+
+_Give developers and operators an easier way to understand the behavior and
```

Comment:
One small trick I like to try in these short synopses is tying this back to the client and/or user. It's ok if that's not 100% possible (ex. if something made our developers 1000% happier at low implementation cost with little/no benefit to the user, we'd just do it), but it's helpful practice for ensuring we're creating clear value. For example, we want to keep/recover service health and allow users to consistently use our applications.

Reply:
I changed the objective slightly to mention a more consistent experience for the user.